src/css/skins/seven.less

```diff
@@ -129,55 +121,63 @@
     /* Styles for play button on idle */
     .jw-display-icon-container {
         border-radius: 50%;
-        border: 1px solid #333;
+        width: 4.5em;
+        height: 4.5em;
+        margin: -2.25em auto 0;
+        
```

Comment:
@robwalch ok, so I got us 80% there. Let's score it so someone more skilled can finish these final "issues."